facebook/delphiFacebook/R/variables.R

```diff
@@ -225,6 +225,8 @@ code_testing <- function(input_data) {
 #' @param input_data input data frame of raw survey data
 #' @return data frame augmented with `v_covid_vaccinated` and
 #'   `v_accept_covid_vaccine`
+#'
+#' @importFrom dplyr coalesce
```

Comment:
you'll want to run `devtools::document()` and commit the `NAMESPACE` change resulting from this as well